intern/openmesheffect/host/intern/properties.c

```diff
@@ -85,86 +94,79 @@ void free_properties(OfxPropertySetStruct *properties) {
-      (0 == strcmp(property, kOfxMeshAttribPropComponentCount) && type == PROP_TYPE_INT) ||
-      (0 == strcmp(property, kOfxMeshAttribPropType) && type == PROP_TYPE_STRING) ||
-      false
-      );
-  case PROP_CTX_OTHER:
-  default:
-    printf("Warning: PROP_CTX_OTHER is depreciated.\n");
-    return true;
+    case PROP_CTX_MESH_EFFECT:
+      return ((0 == strcmp(property, kOfxMeshEffectPropContext) && type == PROP_TYPE_STRING) ||
+              false);
+    case PROP_CTX_INPUT:
+      return ((0 == strcmp(property, kOfxPropLabel) && type == PROP_TYPE_STRING) || false);
+    case PROP_CTX_HOST:
+      return (
+          (0 == strcmp(property, kOfxHostPropBeforeMeshReleaseCb) && type == PROP_TYPE_POINTER) ||
+          (0 == strcmp(property, kOfxHostPropBeforeMeshGetCb) && type == PROP_TYPE_POINTER) ||
+          false);
+    case PROP_CTX_MESH:
+      return ((0 == strcmp(property, kOfxMeshPropInternalData) && type == PROP_TYPE_POINTER) ||
+              (0 == strcmp(property, kOfxMeshPropHostHandle) && type == PROP_TYPE_POINTER) ||
+              (0 == strcmp(property, kOfxMeshPropPointCount) && type == PROP_TYPE_INT) ||
+              (0 == strcmp(property, kOfxMeshPropVertexCount) && type == PROP_TYPE_INT) ||
+              (0 == strcmp(property, kOfxMeshPropFaceCount) && type == PROP_TYPE_INT) || false);
+    case PROP_CTX_PARAM:
+      return ((0 == strcmp(property, kOfxParamPropType) && type == PROP_TYPE_STRING) ||
+              (0 == strcmp(property, kOfxParamPropScriptName) && type == PROP_TYPE_STRING) ||
+              false);
+    case PROP_CTX_ATTRIB:
+      return ((0 == strcmp(property, kOfxMeshAttribPropData) && type == PROP_TYPE_POINTER) ||
+              (0 == strcmp(property, kOfxMeshAttribPropComponentCount) && type == PROP_TYPE_INT) ||
+              (0 == strcmp(property, kOfxMeshAttribPropType) && type == PROP_TYPE_STRING) ||
+              false);
+    case PROP_CTX_OTHER:
+    default:
+      return true;
   }
 }
 
 // // Property Suite Entry Points
 
 const OfxPropertySuiteV1 gPropertySuiteV1 = {
-    /* propSetPointer */   propSetPointer,
-    /* propSetString */    propSetString,
-    /* propSetDouble */    propSetDouble,
-    /* propSetInt */       propSetInt,
-    /* propSetPointerN */  propSetPointerN,
-    /* propSetStringN */   propSetStringN,
-    /* propSetDoubleN */   propSetDoubleN,
-    /* propSetIntN */      propSetIntN,
-    /* propGetPointer */   propGetPointer,
-    /* propGetString */    propGetString,
-    /* propGetDouble */    propGetDouble,
-    /* propGetInt */       propGetInt,
-    /* propGetPointerN */  propGetPointerN,
-    /* propGetStringN */   propGetStringN,
-    /* propGetDoubleN */   propGetDoubleN,
-    /* propGetIntN */      propGetIntN,
-    /* propReset */        propReset,
-    /* propGetDimension */ propGetDimension
-};
-
-OfxStatus propSetPointer(OfxPropertySetHandle properties, const char *property, int index, void *value) {
+    /* propSetPointer */ propSetPointer,
```

Comment:
Is Blender's official clang formatting doing that? Afaik such alignment is present in some of Blender kernel's c files.